README.md

```diff
@@ -30,3 +30,7 @@ Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Con
 ## Contact
 
 You can reach the admins, maintainers, and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the OSP project, you may join the stream [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) and ask questions or interact with the community. Join Us!
+
+## License
+
+Mentorship System is licensed under the GNU General Public License v3.0. Learn more about it in the [LICENSE](LICENSE) file.
```

Comment:
```suggestion
Open Source Programs project is licensed under the GNU General Public License v3.0. Learn more about it in the [LICENSE](LICENSE) file.
```
little copy paste mistep 😉  After this I can approve :)

Reply:
yah sorry @isabelcosta I will revise it 😅